optuna/storages/rdb/storage.py

```diff
@@ -71,6 +71,8 @@ class RDBStorage(BaseStorage):
         engine_kwargs:
             A dictionary of keyword arguments that is passed to
             `sqlalchemy.engine.create_engine`_ function.
+        skip_compatibility_check:
+            Skip to check the schema compatibility if True.
```

Comment:
A nit on the grammar.
```suggestion
            Flag to skip schema compatibility check if set to True.
```

Reply:
Thank you!